src/types/widget.ts

```diff
@@ -11,6 +11,7 @@ interface InitOptions {
   helper: Helper;
   templatesConfig: object;
   createURL(state: SearchParameters): string;
+  onHistoryChange?(state: Partial<SearchParameters>): void;
```

Comment:
`onHistoryChange` is not used by any widget anymore. We can remove it from here and the tests.

Reply:
Oh ok, I thought that it was missing from the interface, I'll remove it from here and the tests.